sentry-rails/spec/sentry/rails/active_support_breadcrumbs_spec.rb

```diff
@@ -88,7 +92,7 @@
         ActiveSupport::Notifications.publish "foo", Object.new
       end.not_to raise_error
 
-      expect(transport.events).to be_empty
+      expect(breadcrumb_buffer.count).to be_zero
```

Comment:
While testing I figure out that this test is not really reliable:
If you run 

```
    it "ignores events that doesn't have a started timestamp" do
      expect do
        ActiveSupport::Notifications.publish "foo", Time.now
      end.not_to raise_error

      expect(transport.events).to be_empty
    end
```

It will pass.
So I use `breadcrumb_buffer` to check if we record or not an event. Maybe we can do better, let me know.